src/main/java/org/jenkinsci/plugins/authorizeproject/AuthorizeProjectStrategyDescriptor.java

```diff
@@ -122,4 +125,16 @@ public boolean isEnabledByDefault() {
     public boolean isApplicableToGlobal() {
         return true;
     }
+
+    protected void prepareNewInstance(StaplerRequest req) {
+
+    }
+
+    /**
+     *
+     */
+    protected void finalizeNewInstance(StaplerRequest req) {
+
+    }

```

Comment:
Yeah I meant to remove them, they were an experiment before I realised reconfigure no longer applied